docs/preview/yamlgettingstarted-directory.md

```diff
@@ -0,0 +1,67 @@
+# YAML getting started - YAML directory (not available, for discussion only)
+
+## Summary
+
+We plan to support a convention-based directory structure, with a couple goals in mind:
+
+1. Today we only create a build definition on-push for the root file `.vsts-ci.yml`. And resource authorization on-push is only handled for that one file. With a convention based
+directory structure, we can watch for files under a specific directory.
+
+2. For template scenarios, a convention-based directory structure would enable referencing templates by name, instead of by relative path.
+
+## Directory structure
+
+```
+.vsts-pipelines/
```

Comment:
Could the folder name be shortened to `.vsts/`?

Reply:
we will probably shorten to .pipelines

Reply:
I agree with @natemcmaster.  What happens if I use multiple build systems and they all use `.pipelines/`?  I would rather the folder be specific to vsts, and have the shortest name possible.

Reply:
FYI - we started using `.vsts/` in aspnet, but for a different reason: we don't want build definitions to be auto-created on push. This spec didn't mention if there would be a way to prevent auto-definition creation, so we picked a folder name that isn't currently in these plans.

Reply:
How many current build systems use/must use .pipelines for their yaml?

Reply:
None that I am aware of currently.  I know that [Gitlab](https://docs.gitlab.com/ee/user/project/pipelines/settings.html#custom-ci-config-path) allows for custom paths.  Still, I would associate `.vsts/` with your product easier than `.pipelines/`